crates/table/src/layout.rs

```diff
@@ -89,7 +88,7 @@ pub trait HasLayout {
 ///   Supporting recursive types remains a TODO(future-work).
 ///   Note that the previous Spacetime datastore did not support recursive types in tables.
 ///
-/// - [`BuiltinType`] is separated into [`PrimitveType`] (atomically-sized types like integers)
+/// - Types where `ty.is_scalar()` are separated into [`PrimitveType`] (atomically-sized types like integers)
 ///   and  [`VarLenType`] (strings, arrays, and maps).
```

Comment:
I'm not sure I would use the term scalar here. Setting strings aside, are arrays and maps considered scalar types? According to the implementation of `is_scalar()`, I don't think they are.

Reply:
Ok, but this comment is saying that strings, arrays, and maps are scalars.